### PR TITLE
Some cleanup of system view scripts

### PR DESCRIPTION
**Pull request by huansong**
A follow-up task from #15239 . Please review each commit individually.

1. Remove gp_toolkit_test.sql 

It seems that this file is used nowhere. Many of the views are outdated.
We have a regress/gp_toolkit test which covers most of what
gp_toolkit_test.sql is supposed to test. So removing gp_toolkit_test.sql,
and adding those exist there but not regress/gp_toolkit to regress/gp_toolkit.

2. Initialize capability of default resource queue in .dat file 

Currently the "cdb_schema.sql" file makes the change to alter the default
resource queue 'pg_default' to have desired capability. More appropriate
way of setting up such initial state should be done via the common catalog
setup which is burning that into a .dat file for the corresponding catalog.
Do that now, and add a regress test case.

3. Further reduce cdb_schema.sql 

Initially a large file that contains all the MPP administrative
views/functions (b9fbacd), over the years cdb_schema.sql
has been reduced to just have a handful of view/functions and
really lost its original purpose. New system views/functions
have been mostly added to system_views.sql or gp_toolkit.sql.
However, the existence of cdb_schema.sql still confused devs
(me for sure) about which file new view/functions should be
added to.

Now further cleanup cdb_shema.sql and add a comment that better
explains the situation. It now only contains a plpgsql function
which cannot be created in system_views.sql and one of its
related function. In fact, looking at the commit that introduced
the plpgsql function (4b86ca7),
it seems that the reason for using plpgsql was a workaround to
some executor/planner problem. If possible we should consider
either fix that problem, or try something like this commit
(d316346) did to a similar problem. After that we could get
rid of cdb_schema.sql as a whole.

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: src/backend/catalog/system_views.sql [hunk 1]
```diff
@@ -1557,6 +1557,44 @@ REVOKE ALL ON pg_subscription FROM public;
 GRANT SELECT (subdbid, subname, subowner, subenabled, subslotname, subpublications)
     ON pg_subscription TO public;
 
+-------------------------------------------------------------------
+-- GPDB legacy cluster-wise segment view
+-------------------------------------------------------------------
+CREATE OR REPLACE VIEW pg_catalog.gp_pgdatabase AS
```
Comment (l-wang) [inline src/backend/catalog/system_views.sql:1563]: All of the views in these file would be created in `pg_catalog` schema correct? Should we remove the `pg_catalog` schema qualifier to make them consistent with the existing DDLs in this file? (I have no strong opinion though and actually like being explicit)

File: src/include/catalog/pg_resqueuecapability.dat [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#----------------------------------------------------------------------
+#
+# pg_resqueuecapability.dat
+#    Initial contents of the pg_resqueuecapability system relation.
+#
+# Portions Copyright (c) 1996-2018, PostgreSQL Global Development Group
+# Portions Copyright (c) 1994, Regents of the University of California
```
Comment (l-wang) [inline src/include/catalog/pg_resqueuecapability.dat:7]: Is this not up to date? Don't we need something with Greenplum and VMware?

**Comment by l-wang** [comment]
It's always nice to remove dead tests and tidy up things! Thank you @huansong for doing it!

File: src/backend/catalog/gp_toolkit_test.sql [hunk 3]
```diff
@@ -1,101 +0,0 @@
-
```
[the remark this answers is not in the document]
Comment (huansong) [reply src/backend/catalog/gp_toolkit_test.sql:1]: I guess it's a little too ancient... Two key commits I could find:
* The initial introduction of the test https://github.com/greenplum-db/gpdb/commit/f1ee02a8508d , for something called "jetpack" which seems to be the old equivalent for gp_toolkit. 
* The addition of the regress test for gp_toolkit: https://github.com/greenplum-db/gpdb/commit/0d2672a6aac7 . As the commit message says:
```
Move the test case from bugbuster into the main suite - this should
not be considered optional or auxiliary anymore.
```

I didn't find when exactly we stopped testing the initial test file even optionally. There were just too many name changes and file moves. But I think we are OK due to the coverage of regress test.

File: src/backend/catalog/system_views.sql (hunk 1 above)
Comment (huansong) [reply src/backend/catalog/system_views.sql:1563]: Yes we can do that. I think it's a good idea to keep consistent. Done.

File: src/include/catalog/pg_resqueuecapability.dat (hunk 2 above)
Comment (huansong) [reply src/include/catalog/pg_resqueuecapability.dat:7]: Ah you are right. Added the Greenplum/VMware statements.

**Review by l-wang — APPROVED** [review]

**Pull request merged**